lib/test_server.py

```diff
@@ -108,9 +109,58 @@ def test_broadcast_message():
 
 # def test_can_add_a_group():
 #   flask_test_client = app.test_client()
-#   group_test = 
+#   group_test =
+
+def test_active_sockets():
+  Connection.objects.delete()
+
+  assert len(Connection.objects(group='test2')) == 0
+
+  flask_test_client = app.test_client()
+  socketio_test_client = socketio.test_client(app, flask_test_client=flask_test_client)
+  socketio_test_client.emit('join_group', {'access_code': "test2"})
+  conn1 = Connection.objects.get(sid=socketio_test_client.sid)
+
+  assert len(Connection.objects(group='test2')) == 1
+  assert conn1.sid == socketio_test_client.sid
+
+  socketio_test_client2 = socketio.test_client(app, flask_test_client=flask_test_client)
+  socketio_test_client2.emit('join_group', {'access_code': "test2"})
+  conn2 = Connection.objects.get(sid=socketio_test_client2.sid)
+
+  assert len(Connection.objects(group='test2')) == 2
+  assert conn2.sid == socketio_test_client2.sid
+
+  socketio_test_client.disconnect()
+
+  assert len(Connection.objects(group='test2')) == 1
+
+  socketio_test_client2.disconnect()
+
+  assert len(Connection.objects(group='test2')) == 0
+
+def test_group_connection_differentiation():
```

Comment:
Added this test to make sure Connections across separate groups increment and decrement as expected

Reply:
I like that you tested it incrementally for both increase and decrease connection collection size